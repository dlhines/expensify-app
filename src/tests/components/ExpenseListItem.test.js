import React from 'react';
import { shallow } from 'enzyme';
import expenses from '../fixtures/expenses';
import ExpenseListItem from '../../components/ExpenseListItem';

test('should render snapshot ExpenseListItem with array data', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]} />);
    expect(wrapper).toMatchSnapshot();
});