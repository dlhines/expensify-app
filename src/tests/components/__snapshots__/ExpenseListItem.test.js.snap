// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot ExpenseListItem with array data 1`] = `
<div>
  <h3>
    <Link
      replace={false}
      to="/edit/1"
    >
      Goober
    </Link>
  </h3>
  <p>
    $10.95
     - 
    December 31st, 1969
  </p>
</div>
`;
